numpy/random/mtrand/mtrand.pyx

```diff
@@ -2201,7 +2201,7 @@ cdef class RandomState:
         Parameters
         ----------
         df : int
-            Degrees of freedom, should be >= 1.
+            Degrees of freedom, should be > 0.
```

Comment:
Should note that `df == 1` is only available in numpy 1.10 or greater. Hmm... maybe something like
```
Degrees of freedom, must be > 0 as of NumPy 1.10,
must be > 1 for earlier versions.
```